app/main.py

```diff
@@ -1,3 +1,71 @@
-def shop_trip():
-    # write your code here
-    pass
+import json
+import datetime
+
+from app.car import Car
+from app.shop import Shop
+from app.customer import Customer
+
+
+def shop_trip() -> None:
+    with open(
+            "app/config.json",
+            "rb"
+    ) as datafile:
+        data = json.load(datafile)
+
+    fuel_price = data["FUEL_PRICE"]
+    customers = data["customers"]
+    shops = data["shops"]
+
+    for customer in customers:
+        buyer = Customer(**customer)
+        buyer.car = Car(**customer["car"])
+        print(f"{buyer.name} has {buyer.money} dollars")
+        road_prices = {}
+
+        for shop in shops:
+            market = Shop(**shop)
+            road_price = market.calc_trip_price(
+                customer_location=buyer.location,
+                fuel_price=fuel_price,
+                fuel_consumption=buyer.car.fuel_consumption,
+                customer_cart=buyer.product_cart)
```

Comment:
Your closing parenthesis should be placed on a new line and should not contain any arguments to match the opening parenthesis code style.